pubsub/Gemfile

```diff
@@ -14,7 +14,7 @@
 
 source "https://rubygems.org"
 
-gem "google-cloud-pubsub"
+gem "google-cloud-pubsub", "~> 2.2"
```

Comment:
@TheRoyalTnetennba Should I not specify a version constraint for `google-cloud-pubsub` in the Gemfile? (For the example added in this PR, version `2.2` or above is required.)

Reply:
I think it is good to specify version constraints where required.